components/RadioPlayer.vue

```diff
@@ -7,6 +7,14 @@
 
       <div class="now-playing-details">
         <div class="radio-controls">
+          <div v-if="showAlbumArt && np.now_playing.song.art" class="now-playing-art">
+            <a class="cursor-pointer programimage" rel="playerimg" @click.stop="streamModal = !streamModal">
+              <div v-if="show_check == true" class="onair">On air</div>
+              <img class="progimg" :src="show_art_url" :alt="'album_art_alt'">
+            </a>
+            <Modal :media="show_art_url" :title="show_title" :description="show_subtitle" :visibility="streamModal" @close="closeModal" />
+          </div>
+
           <div class="play-volume-controls">
```

Comment:
This is again maybe the wrong place -- let me know if we should move this discussion just say please create a separate issue for this, dude and I will do so -- 

Do you think we should add skip-backward skip-forward buttons? Especially for arcsi items that would be useful to be able to use them instead of seeking by hand

Reply:
good idea, but first let's decide if arcsi player is also here